features/config.feature

```diff
@@ -534,3 +534,14 @@ require_once(ABSPATH . 'wp-settings.php');
       """
       http://example.com
       """
+
+  Scenario: If there is a wrong value defined for `DOMAIN_CURRENT_SITE` constant
```

Comment:
This scenario will need to define an invalid value for `DOMAIN_CURRENT_SITE` in order for the error to occur...